superfluidDaoFront/src/App.tsx

```diff
@@ -0,0 +1,21 @@
+import "./App.css";
+import NavBar from "./components/NavBar";
+import LandingComponent from "./pages/Landing";
+import CreateDao from "./pages/CreateDao";
+import YoursDao from "./pages/YoursDao";
+import { Routes, Route } from "react-router-dom";
+
+function App() {
+  return (
+    <>
+      <NavBar />
+      <Routes>
```

Comment:
A better way should be to use a `Layout` architecture to add the `Navbar` to each page and a future `Footer` I guess.
see https://reactrouter.com/en/main/start/concepts#layout-routes